modules/user_manual/pages/apps/activity.adoc

```diff
@@ -55,5 +55,5 @@ image::apps/activity/activity-settings-limit-to-favorites.png[Limit activity set
 
 === Configuring the Email Notification Interval
 
-In addition to enabling and disabling email notifications, bulk email notifications can be configured to be sent out hourly, daily, or weekly.
+In addition to enabling and disabling email notifications, bulk email notifications can be configured to be sent out: _As soon as possible_, _Hourly_, _Daily_, and _Weekly_.
```

Comment:
May be add a clarification that 'As soon as possible' means the next  cron execution?